src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp

```diff
@@ -15,22 +15,39 @@
 #include "observer.hpp"
 #include "routerSubscriber.hpp"
 #include "singleton.hpp"
+#include "stringHelper.h"
 #include <external/nlohmann/json.hpp>
 #include <functional>
 #include <memory>
 #include <string>
+#include <unordered_set>
 #include <vector>
 
 /**
  * @brief PolicyManager class.
  *
  */
+
+constexpr auto UNKNOWN_VALUE {" "};
 class PolicyManager final : public Singleton<PolicyManager>
 {
 private:
     Subject<nlohmann::json&> m_subject;
     nlohmann::json m_configuration;
     std::unique_ptr<RouterSubscriber> m_policyChangeSubscription;
+    bool m_vulDetectEnabled {true};
+    bool m_indexerEnabled {true};
+    bool m_vulDetectValid {true};
+    bool m_indexerValid {true};
+    long m_feedUpdateTime {300};
+    std::string m_urlFeed {UNKNOWN_VALUE};
+    std::unordered_set<std::string> m_hostList;
+    std::unordered_set<std::string> m_caList;
+    std::string m_username {UNKNOWN_VALUE};
+    std::string m_password {UNKNOWN_VALUE};
+    std::string m_certificate {UNKNOWN_VALUE};
+    std::string m_key {UNKNOWN_VALUE};
```

Comment:
What is the use of these private member variables if they don't have getters to be obtained from outside the class?
How are they going to be used?

Reply:
I don't know if this is included in the scope of this PR. For now, we only validate the JSON fields, and possibly these fields will be used only in this class so that getters won't be necessary.

Reply:
Done =)